lib/os.js

```diff
@@ -22,6 +22,9 @@ exports.platform = function() {
   return process.platform;
 };
 
+const trailingSlashRe = isWindows ? /[^:]\\$/
```

Comment:
this seems like something that should live in path.js, it'd be nice if we could confine a lot of this platform path logic to one place and export it to wherever it's needed